README.md

```diff
@@ -35,17 +35,14 @@ You can learn more about the story behind Cartography in our [presentation at BS
 Start [here](docs/setup/install.md).
 
 ## Supported platforms
-- [Amazon Web Services](docs/setup/config/aws.md) -  API Gateway, Config, EC2, Elasticsearch, Elastic Kubernetes Service, DynamoDB, IAM, KMS, Launch Configurations, Launch Templates, Lambda, RDS, Redshift, Route53, S3, Secrets Manager, Security Hub, SQS, STS, Tags
```

Comment:
There seems to be some changes to AWS services supported here. Can you revert back the documentation changes to AWS here?

Reply:
I think you need to merge latest master branch into your branch.